site/docs/fileio.md

```diff
@@ -30,15 +30,11 @@ The metadata for an Iceberg table tracks the absolute path for data files which
 
 ## Usage in Processing Engines
 
-The responsibility of reading and writing data files lies with the processing engines and happens during task execution. However, after data files are written, processing engines use FileIO to write new Iceberg metadata files that capture the new state of the table. A blog post that provides a deeper understanding of FileIO is
-[Iceberg FileIO: Cloud Native Tables](https://tabular.io/blog/iceberg-fileio/)
```

Comment:
@Fokko Do you happen to know any new links for these expired ones?

Reply:
Let me ask internally, I think the FileIO would be a good one to add to the website 👍